.github/workflows/docker.yml

```diff
@@ -13,6 +17,9 @@ defaults:
 env:
   DOCKER_ORG: lightninglabs
   DOCKER_REPO: lightning-terminal
+  NIGHTLY_DOCKER_REPO: lightning-terminal-nightly
+  NIGHTLY_BRANCH_NAME: 0-19-staging
```

Comment:
i assume we will eventually change this to master?

Reply:
Correct. Currently we get the most value out of nightly builds of the experimental branch, because that's where we iterate the fastest. Once that's in master, we'll change this too.
Going to add a TODO to make it clear.